lib/stat_tracker_dummy_initiator.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './stat_tracker'
+
+class StatTrackerDummyInitiator
+  def self.create
+    game_path = './data/game_dummy.csv'
+    team_path = './data/team_info.csv'
+    game_teams_path = './data/game_teams_stats_dummy.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    StatTracker.from_csv(locations)
+  end
+end
```

Comment:
We can use this class variable to pass "dummy" files we created will much less data. We can use this to create GameStats methods without having to use ALL of the data, and only what we need.